Extension/package.json

```diff
@@ -2763,7 +2762,6 @@
         "configurationAttributes": {
           "launch": {
             "type": "object",
-            "default": {},
```

Comment:
If updating the debug configuration, you should also update `tools/OptionsSchema.json`.  Remove the `default: {}` from "CppvsdbgLaunchOptions" and "CppdbgLaunchOptions"